tests/libs/datasets/outlier_detection_test.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+from covidactnow.datapublic.common_fields import CommonFields
+from tests import test_helpers
+from libs.datasets import outlier_detection
+from libs.datasets.taglib import TagType
+
+TimeseriesLiteral = test_helpers.TimeseriesLiteral
+
+
+def test_remove_outliers():
+    values = [10.0] * 7 + [1000.0]
+    dataset = test_helpers.build_default_region_dataset({CommonFields.NEW_CASES: values})
+    dataset = outlier_detection.drop_new_case_outliers(dataset)
+
+    # Expected result is the same series with the last value removed
+    expected_tag = test_helpers.make_tag(
+        TagType.ZSCORE_OUTLIER, date="2020-04-08", original_observation=1000.0,
+    )
+    expected_ts = TimeseriesLiteral([10.0] * 7, annotation=[expected_tag])
+    expected = test_helpers.build_default_region_dataset({CommonFields.NEW_CASES: expected_ts})
+    test_helpers.assert_dataset_like(dataset, expected, drop_na_dates=True)
+
+
+def test_remove_outliers_threshold():
+    values = [1.0] * 7 + [30.0]
+    dataset = test_helpers.build_default_region_dataset({CommonFields.NEW_CASES: values})
+    result = outlier_detection.drop_series_outliers(dataset, CommonFields.NEW_CASES, threshold=30)
+
+    # Should not modify becasue not higher than threshold
+    test_helpers.assert_dataset_like(dataset, result)
+
+    result = outlier_detection.drop_series_outliers(dataset, CommonFields.NEW_CASES, threshold=29)
+
+    # Expected result is the same series with the last value removed
+    expected_tag = test_helpers.make_tag(
+        TagType.ZSCORE_OUTLIER, date="2020-04-08", original_observation=30.0
+    )
+    expected_ts = TimeseriesLiteral([1.0] * 7, annotation=[expected_tag])
+    expected = test_helpers.build_default_region_dataset({CommonFields.NEW_CASES: expected_ts})
+    test_helpers.assert_dataset_like(result, expected, drop_na_dates=True)
+
+
+def test_not_removing_short_series():
+    values = [None] * 7 + [1, 1, 300]
+    dataset = test_helpers.build_default_region_dataset({CommonFields.NEW_CASES: values})
+    result = outlier_detection.drop_series_outliers(dataset, CommonFields.NEW_CASES, threshold=30)
+
+    # Should not modify becasue not higher than threshold
```

Comment:
may as well fix this while copying it, maybe by removing the comment because the test name is pretty clear